features/wbr.yml

```
@@ -1,6 +1,5 @@
-draft_date: 2024-06-05
 name: <wbr>
-description: The `<wbr>` element TODO.
+description: The `<wbr>` HTML element represents a word break opportunity.
```

Do you think it'd be excessive to elaborate on this?

```suggestion
description: The `<wbr>` HTML element represents a word break opportunity, to explicitly mark a place within a word where a line might be broken.
```